solver/app/testdata/TestTokens.golden

```diff
@@ -479,6 +479,16 @@
   "name": "Hyperliquid",
   "symbol": "HYPE"
  },
+ {
+  "address": "6AfTP38VbvM3gxMRy8ubvb7NbhzhjVB8VGMy2uBby1pY",
```

Comment:
Shouldn't the SVM address also be present here?

Reply:
that is the svm address